integration_tests/e2e/goal/unarchiveGoal.cy.ts

```diff
@@ -22,6 +29,7 @@ context('Unarchive a goal', () => {
     cy.task('getPrisonerById')
     cy.task('stubGetInduction')
     cy.task('getActionPlan')
+    cy.task('getGoalsByStatus')
```

Comment:
As above, this sets up stubs for both `getActionPlan` and `getGoalsByStatus`. Is that right? 🤔